src/cppwinrt.build.pre.props

```diff
@@ -11,6 +11,7 @@
   <Import Project="$(OpenConsoleDir)\packages\Microsoft.Windows.CppWinRT.2.0.210309.3\build\native\Microsoft.Windows.CppWinRT.props" Condition="Exists('$(OpenConsoleDir)\packages\Microsoft.Windows.CppWinRT.2.0.210309.3\build\native\Microsoft.Windows.CppWinRT.props')" />
 
   <PropertyGroup Label="Globals">
+    <CppWinRTHeapEnforcement>AnyValueHereWillDisableTheOptOut</CppWinRTHeapEnforcement>

```

Comment:
Based on your PR description, I'm not following why this is required as well.

Reply:
Oh, sorry. C++/WinRT disables MAKE_DETECTION if _any xaml is present_ and this variable is unset.

Reply:
Weird. OK.